src/helpers/assertStringsWithinText.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactTestRendererNode } from 'react-test-renderer';
+
+export const assertStringsWithinText = (
+  rendererJSON: ReactTestRendererNode | null | ReactTestRendererNode[]
```

Comment:
```suggestion
  node: ReactTestRendererNode | ReactTestRendererNode[] | null
```

Reply:
I wonder if we should accept just `ReactTestRendererNode | null` or even `ReactTestRendererNode`? Do we really need `ReactTestRendererNode[]`?

Reply:
The return type of the toJson function is TestRenderer.ReactTestRendererJSON | TestRenderer.ReactTestRendererJSON[] | null, so we need it, I refactor the assertStringsWithinText a bit to break it in two so the first one handles all types one the second one only accepts a ReactTestRendererNode, I think it's clearer that way, is it ok for you ?